tests/vmi_lifecycle_test.go

```diff
@@ -569,6 +569,92 @@ var _ = Describe("[rfe_id:273][crit:high][vendor:cnv-qe@redhat.com][level:compon
 
 				tests.WaitForSuccessfulVMIStart(newVMI)
 			})
+
+			It("[test_id:5275]VMIs with Bridge Networking shouldn't fail after the kubelet restarts", func() {
+				bridgeVMI := vmi
+				// Remove the masquerade interface to use the default bridge one
+				bridgeVMI.Spec.Domain.Devices.Interfaces = nil
+				bridgeVMI.Spec.Networks = nil
+				v1.SetDefaults_NetworkInterface(bridgeVMI)
+				Expect(bridgeVMI.Spec.Domain.Devices.Interfaces).NotTo(BeEmpty())
+
+				By("starting a VMI with bridged network on a node")
+				bridgeVMI, err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(bridgeVMI)
+				Expect(err).To(BeNil(), "Should submit VMI successfully")
+
+				// Start a VirtualMachineInstance with bridged networking
+				nodeName := tests.WaitForSuccessfulVMIStart(bridgeVMI)
+
+				tests.VerifyDummyNicForBridgeNetwork(bridgeVMI)
+
+				By("restarting kubelet")
+				pod := renderPkillAllPod("kubelet")
+				pod.Spec.NodeName = nodeName
+				_, err = virtClient.CoreV1().Pods(tests.NamespaceTestDefault).Create(pod)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("starting another VMI on the same node, to verify kubelet is running again")
+				newVMI := newCirrosVMI()
+				newVMI.Spec.NodeSelector = map[string]string{"kubernetes.io/hostname": nodeName}
+				Eventually(func() error {
+					newVMI, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(newVMI)
+					Expect(err).To(BeNil())
+					return nil
+				}, 100, 10).Should(Succeed(), "Should be able to start a new VM")
+
+				By("checking if the VMI with bridged networking is still running, it will verify the CNI didn't cause the pod to be killed")
+				tests.WaitForSuccessfulVMIStart(bridgeVMI)
+			})
+
+			It("[test_id:5276]VMIs with Bridge Networking should work with Duplicate Address Detection (DAD)", func() {
```

Comment:
Where did you take this and the other test number from?

Reply:
I looked for the highest test number then added 1

Reply:
I wonder why there is no vendor tag, indicating that Red Hat QE manages them. These tags are "owned" by Red Hat QE. You can just ignore them and don't have to invent some. They will add them if they want/need.